File: Basics/Conditional-Statements-Advanced/Lab/working_hours.py
hour = int(input())
day = input()

working = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]

if day in working:
    if 10 <= hour <= 18:
        print(f"open")
    else:
        print(f"closed")
else:
    print(f"closed")

File: JS-Basics/0-3-Conditional-Steatements-Advanced/Lab/working-hours.js
function isOfficeWorking(input) {
    let hour = Number(input[0]);
    let day = input[1];
    let workingDays = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
    let workingHours = [10, 11, 12, 13, 14, 15, 16, 17, 18];
    if (workingDays.includes(day) && workingHours.includes(hour)) {
        console.log("open");
    } else {
        console.log("closed");
    }
}
